src/flb_log.c

```diff
@@ -704,8 +721,11 @@ void flb_log_print(int type, const char *file, int line, const char *fmt, ...)
     int ret;
     struct log_message msg = {0};
     va_list args;
+    char *msg_type_str = flb_log_message_type_str(type);
```

Comment:
Same as with `flb_log_construct`, please decouple the declaration from the initialization of both `msg_type_str` and `ts`

Reply:
Done.